Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -280,6 +283,28 @@ - (void)adjustFrameForSafeInsets
     _noteEditorTextView.frame = viewFrame;
 }
 
+- (void)handleNoteEditorDidBecomeFirstResponderNotification:(NSNotification *)notification {
+    
+    checklistButton.hidden = NO;
```

Comment:
(Not introduced in your PR!!) but this one should probably be a @property, WDYT?

Reply:
I agree the NSButtons should definitely be properties on the class. I'd probably handle orthogonal changes like that via a separate PR but I can add there here too.